osu.Game.Tests/Mods/ModSettingsTest.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Osu.Mods;
+
+namespace osu.Game.Tests.Mods
+{
+    public class ModSettingsTest
+    {
+        [Test]
+        public void TestModSettingsUnboundWhenCopied()
+        {
+            var original = new OsuModDoubleTime();
+            var copy = (OsuModDoubleTime)original.DeepClone();
+
+            original.SpeedChange.Value = 2;
+
+            Assert.That(original.SpeedChange.Value, Is.EqualTo(2.0));
+            Assert.That(copy.SpeedChange.Value, Is.EqualTo(1.5));
```

Comment:
Side note: while the original test used `Precision.AlmostEquals()` for these assertions, that was 99% overkill. `2.0` and `1.5` are both perfectly presentable floating-point numbers and there should be no representation error of them, hence no need for `Precision.AlmostEquals()`.